diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in -x m4 origsrc/libgalago-gtk-0.5.0/Makefile.am src/libgalago-gtk-0.5.0/Makefile.am
--- origsrc/libgalago-gtk-0.5.0/Makefile.am	2006-02-13 02:18:46.000000000 -0600
+++ src/libgalago-gtk-0.5.0/Makefile.am	2007-06-22 06:46:50.265625000 -0500
@@ -1,4 +1,4 @@
-SUBDIRS = autopackage intl libgalago-gtk pixmaps po tests
+SUBDIRS = autopackage libgalago-gtk pixmaps po tests
 
 pcdata_DATA= libgalago-gtk.pc
 pcdatadir = $(libdir)/pkgconfig
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in -x m4 origsrc/libgalago-gtk-0.5.0/configure.ac src/libgalago-gtk-0.5.0/configure.ac
--- origsrc/libgalago-gtk-0.5.0/configure.ac	2006-04-08 21:52:17.000000000 -0500
+++ src/libgalago-gtk-0.5.0/configure.ac	2007-06-22 06:46:39.531250000 -0500
@@ -88,8 +88,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [gettext package])
 
-AM_GNU_GETTEXT_VERSION(0.10.40)
-AM_GNU_GETTEXT
+AM_GLIB_GNU_GETTEXT
 
 pkg_modules="glib-2.0 >= 2.4.0, gtk+-2.0 >= 2.4.0, libgalago >= 0.5.0, dbus-1 >= 0.36, dbus-glib-1 >= 0.36"
 PKG_CHECK_MODULES(PACKAGE, [$pkg_modules])
@@ -118,7 +117,6 @@
 Makefile
 autopackage/Makefile
 autopackage/default.apspec
-intl/Makefile
 libgalago-gtk/Makefile
 pixmaps/Makefile
 po/Makefile.in
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in -x m4 origsrc/libgalago-gtk-0.5.0/libgalago-gtk/Makefile.am src/libgalago-gtk-0.5.0/libgalago-gtk/Makefile.am
--- origsrc/libgalago-gtk-0.5.0/libgalago-gtk/Makefile.am	2006-03-17 01:28:01.000000000 -0600
+++ src/libgalago-gtk-0.5.0/libgalago-gtk/Makefile.am	2007-06-21 23:36:48.031250000 -0500
@@ -63,7 +63,7 @@
 libgalago_gtk_la_LIBADD = $(BINRELOC_LIBS) $(PACKAGE_LIBS)
 
 libgalago_gtk_la_LDFLAGS = \
-	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
+	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -no-undefined
 
 #$(MARSHAL_FILE).h: $(MARSHAL_FILE).list
 #	($(GLIB_GENMARSHAL) --prefix=$(MARSHAL_PREFIX) $(MARSHAL_FILE).list \
