--- origsrc/libgalago-gtk-0.5.0/Makefile.am	2006-02-13 02:18:46.000000000 -0600
+++ src/libgalago-gtk-0.5.0/Makefile.am	2007-06-22 06:46:50.265625000 -0500
@@ -1,4 +1,4 @@
-SUBDIRS = autopackage intl libgalago-gtk pixmaps po tests
+SUBDIRS = autopackage libgalago-gtk pixmaps po tests
 
 pcdata_DATA= libgalago-gtk.pc
 pcdatadir = $(libdir)/pkgconfig
--- origsrc/libgalago-gtk-0.5.0/configure.ac	2006-04-08 21:52:17.000000000 -0500
+++ src/libgalago-gtk-0.5.0/configure.ac	2007-06-22 06:46:39.531250000 -0500
@@ -88,8 +88,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [gettext package])
 
-AM_GNU_GETTEXT_VERSION(0.10.40)
-AM_GNU_GETTEXT
+AM_GLIB_GNU_GETTEXT
 
 pkg_modules="glib-2.0 >= 2.4.0, gtk+-2.0 >= 2.4.0, libgalago >= 0.5.0, dbus-1 >= 0.36, dbus-glib-1 >= 0.36"
 PKG_CHECK_MODULES(PACKAGE, [$pkg_modules])
@@ -118,7 +117,6 @@
 Makefile
 autopackage/Makefile
 autopackage/default.apspec
-intl/Makefile
 libgalago-gtk/Makefile
 pixmaps/Makefile
 po/Makefile.in
--- origsrc/libgalago-gtk-0.5.0/libgalago-gtk/Makefile.am	2006-03-17 01:28:01.000000000 -0600
+++ src/libgalago-gtk-0.5.0/libgalago-gtk/Makefile.am	2007-06-21 23:36:48.031250000 -0500
@@ -63,7 +63,7 @@
 libgalago_gtk_la_LIBADD = $(BINRELOC_LIBS) $(PACKAGE_LIBS)
 
 libgalago_gtk_la_LDFLAGS = \
-	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
+	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -no-undefined
 
 #$(MARSHAL_FILE).h: $(MARSHAL_FILE).list
 #	($(GLIB_GENMARSHAL) --prefix=$(MARSHAL_PREFIX) $(MARSHAL_FILE).list \
